src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -583,6 +583,18 @@ internal static byte[] ConvertIntToByteArray(int dwInput)
             return rgbOutput;
         }
 
+        internal static int ConvertByteArrayToInt(byte[] input)
```

Comment:
I also considered `BitConverter.GetBytes` with `Array.Reverse` but then I read the spec and they seem to indicate that it's closer to `BigInteger` (in that it skips leading 0-value bytes.